BUILD.bazel

```
@@ -0,0 +1,188 @@
+load(
+    "//ign_bazel:build_defs.bzl",
+    "IGNITION_ROOT",
+    "IGNITION_VISIBILITY",
+    "cmake_configure_file",
+    "generate_include_header",
+    "generate_yaml",
+    "ign_config_header",
+    "ign_export_header",
+)
+load(
+    ":ign_msg_gen.bzl",
+    "get_proto_headers",
+    "ign_msg_gen",
+)
+
+package(
+    default_visibility = IGNITION_VISIBILITY,
+    features = [
+        "-parse_headers",
+        "-layering_check",
+    ],
+)
+
+licenses(["notice"])
+
+exports_files(["LICENSE"])
+
+PROJECT_NAME = "ignition-msgs"
+
+PROJECT_MAJOR = 7
+
+PROJECT_MINOR = 0
```

https://github.com/ignitionrobotics/ign-msgs/blob/3d8d8281deb002691db5891fb06d654640027369/CMakeLists.txt#L6

```suggestion
PROJECT_MINOR = 1
```